docgen/src/guide/React_native.md

```diff
@@ -6,9 +6,643 @@ category: guide
+  Text,
+  TextInput,
+  TouchableHighlight,
+} from 'react-native';
+
+// [...]
+
+const RefinementList = connectRefinementList(({ refine, items }) => {
+  const keyExtractor = (item, index) => index;
+
+  return (
+    <FlatList
+      data={items}
+      keyExtractor={keyExtractor}
+      ListHeaderComponent={() =>
+        <Text style={{ marginTop: 20, height: 50, alignSelf: 'center' }}>
+          Categories
+        </Text>}
+      renderItem={({ item }) => {
+        return (
+          <View style={{ height: 30 }}>
+            <TouchableHighlight
+              onPress={() => {
+                refine(item.value);
+              }}
+            >
+              <Text style={item.isRefined ? { fontWeight: 'bold' } : {}}>
+                {item.label}
+              </Text>
+            </TouchableHighlight>
+          </View>
+        );
+      }}
+    />
+  );
+});
+```
+
+Let’s add this new `<RefinementList>` component to our InstantSearch instance. 
+
+```jsx
+const App = () =>
+   <View style={styles.container}>
+        <InstantSearch
+            appId="latency"
+            apiKey="6be0576ff61c053d5f9a3225e2a90f76"
+            indexName="ikea"
+         >
+            <View
+            style={{
+                flexDirection: 'row',
+            }}
+            >
+            <SearchBox />
+            </View>
+            <RefinementList attributeName="category" />
+            <Hits />
+        </InstantSearch>
+    </View>
+```
+
+The `attributeName` props specifies the faceted attribute to use in this widget. This attribute should be declared as a facet in the index configuration as well.
+
+The values displayed are computed by Algolia from the results.
+
+So far, we've seen the following:
+- there are components for all types of refinements
+- the RefinementList works with facets
+- facets are computed from the results
+
+## Putting our Refinement List in a Modal
+Because we don’t have that much space available on our mobile application, we are going to extract this RefinementList and display it inside a [Modal](http://facebook.github.io/react-native/releases/0.45/docs/modal.html#modal).
+
+When using the [Modal](http://facebook.github.io/react-native/releases/0.45/docs/modal.html#modal), components that are displayed inside it are mounted and unmounted depending if the [Modal](http://facebook.github.io/react-native/releases/0.45/docs/modal.html#modal) is visible or not. One thing to know about React InstantSearch is that a refinement is applied only if its corresponding widget is mounted. If a widget is unmounted then its state is removed from the search state as well. 
+
+To keep track of the refinements applied inside the [Modal](http://facebook.github.io/react-native/releases/0.45/docs/modal.html#modal) (or another screen if you have navigation), you will need to use another `<InstantSearch/>` instance and synchronise the search state between those two instances. 
+
+ `<InstantSearch/>` takes two interesting props: 
```

Comment:
Earlier you spelled this as `<InstantSearch>`, now as `<InstantSearch/>`. I think the former is the better option.